test/test.h

```diff
@@ -46,4 +54,16 @@ namespace test
 		Options(Options const&) = delete;
 	};
 
+	class TestOutputHelper
+	{
+	public:
+		TestOutputHelper() { TestOutputHelper::initTest(); }
+		static void initTest();
+		static bool passTest(std::string& _testName);
+		static std::string const& testName() { return m_currentTestName; }
+		static std::string const& caseName() { return m_currentTestCaseName; }
+	private:
+		static std::string m_currentTestName;
```

Comment:
Boost actually creates a new instance of the TestOutputHelper for each test, so no need to use static here and in any other place actually.